app/src/test/java/backgammon/DoublingCubeTests.java

```diff
@@ -0,0 +1,16 @@
+package backgammon;
+
```

Comment:
This doesn't import the necessary JUnit statements. Look into the other files in `app/src/test/java/backgammon` for the import statements.